roles/ha/README.md

```diff
@@ -0,0 +1,28 @@
+# mila.proxmox.ha
+
+To manage HA Groups assignations of VMs or CTs , use the role `mila.proxmox.ha`.
+
+You must provide the `proxmox_ha_resources` variable in order to assign
+ressources to existing HA Groups.
+
+Example:
+
+    ---
+    proxmox_ha_resources:
+      - group: "HA_all"
+        sid: "vm:110"
+      - group: "HA_vn-c"
+        sid: "vm:108"
```

Comment:
Just to be sure my understanding is correct: if one needs to configure 12 VMs in a single group, the structure would be the following?
```
proxmox_ha_resources:
  - group: "HA_group"
    sid: "vm:1"
  - group: "HA_group"
    sid: "vm:2"
  [...]
  - group: "HA_group"
    sid: "vm:12"
```

If so, could we consider a structure with group name and members?

```
proxmox_ha_resources:
  - group: "HA_group"
    members:
      - "vm-1"
      - "vm-2" 
      [...]
      - "vm-12"

Reply:
@pbouchez @ahmam any thoughts on what should be an appropriate structure here?

Reply:
based on the "This role DOES NOT creates the HA Groups! You must create them yourself." I see that the idea of having a group structure members will be the most appropriate in this context.

Reply:
+1 for the group structure members

Reply:
Done !